tests/manager/service/analytics/test_analytics.py

```diff
@@ -48,53 +31,3 @@ def test_init_analytics(self):
         assert len(analytics.providers) == 2
         assert type(analytics.providers[0]) == LocalAnalyticsProvider
         assert type(analytics.providers[1]) == S3AnalyticsProvider
-
-    def test_user_agent_capture(
```

Comment:
The user agent logic was moved into `AnalyticsEventData.create`, so this test was moved into `test_eventdata.py`.